src/app/game/settings/account/change/email/page.tsx

```diff
@@ -0,0 +1,72 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import NavigationFooter from "@/features/game/NavigationFooter";
+import React from "react";
+import { ChevronLeft } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+export default function GameSettingsAccountDelete() {
```

Comment:
こちら、以下のように修正をお願いします。
```suggestion
export default function GameSettingsAccountChangeEmail() {
```
Issue17で取り上げてからコントリビューションガイドに追記してなかったので、申し訳ないですがお願いします。

https://github.com/KobayaClanguage/find-the-object-game-v2/pull/26#discussion_r2019696233